src/main/java/edu/virginia/lib/indexing/ASpaceObject.java

```diff
@@ -515,4 +523,100 @@ public void printOutRawData() {
         JsonHelper.writeOutJson(record);
     }
 
+    public void writeCirculationRecord(File file) throws IOException {
+    	//make MARC record with 245 and 590 fields
+		MarcFactory factory = MarcFactory.newInstance();
+		Record r = factory.newRecord();
+		DataField df;
+		Subfield sf;
+		
+		String title = record.getString("title");
```

Comment:
You probably don't want to use the shared "record" variable here because typically ASpaceObject or descendant classes are provided the id that doesn't include "/tree".  Instead you should add a method to ASpaceObject called getTree() that returns the tree.  Earlier code [here](https://github.com/uvalib/archivesspace-virgo/blob/master/src/main/java/edu/virginia/lib/indexing/ASpaceObject.java#L402-L404) does the same thing we want.

Since the call to get the "tree" is so costly we should ensure its only done once per instance of an ASpaceObject.  If you create a private member variable that is a JsonObject called "tree" that is [lazy loaded](https://en.wikipedia.org/wiki/Lazy_loading) in response to a "getTree()" method and replace your reference and the I linked to to calls to "getTree()" that would achieve the goal.